src/main.cpp

```diff
@@ -16,23 +16,23 @@ Color ray_color(const Ray &r, Hitable *world, int depth);
 
 int main()
 {
-    Image image("test_camera_defocus.ppm");
+    Image image("test_dielectrics.ppm");
 
     auto samples = 8;
     
     Hitable *list[4];
 
     std::size_t i = 0;
     list[i++] = new Sphere(Vec3(0.0f, -1000.0f, 0.0f), 1000.0f - 0.1f, new Lambertian(Color(0.8f, 0.3f, 0.3f)));
-    list[i++] = new Sphere(Vec3(0.0f, 0.0f, -1.0f), 0.5f, new Lambertian(Color(0.8f, 0.8f, 0.0f)));
+    list[i++] = new Sphere(Vec3(0.0f, 0.0f, -1.0f), 0.5f, new Lambertian(Color(0.1f, 0.2f, 0.5f)));
     list[i++] = new Sphere(Vec3(1.0f, 0.0f, -1.0f), 0.5f, new Metal(Color(0.8f, 0.6f, 0.2f), 0.3f));
-	list[i++] = new Sphere(Vec3(-1.0f, 0.0f, -1.0f), 0.5f, new Metal(Color(0.8f, 0.8f, 0.8f), 0.5f));
+	list[i++] = new Sphere(Vec3(-1.0f, 0.0f, -1.0f), 0.5f, new Dielectric(1.5f));
```

Comment:
Fix tabulation, please

Reply:
ehm.. it's already 4 spaces, don't know why it's doing this